conans/tools.py

```diff
@@ -11,11 +11,18 @@
 from conans.client.output import ConanOutput
 
 
+def vcvars(settings):
```

Comment:
maybe call it vcvars_command? If the name is vcvars I could expect that it returns some vars